examples/docs_beta_snippets/docs_beta_snippets_tests/snippet_checks/guides/components/my-existing-project/setup.py

```diff
@@ -0,0 +1,8 @@
+from setuptools import find_packages, setup
```

Comment:
Why is there both `setup.py` and `pyproject.toml` with redundant package metadataa?